lib/iris/plot.py

```diff
@@ -127,8 +126,6 @@ def get_span(coord):
     # Note the use of `reversed` to convert from the X-then-Y
     # convention of the end-user API to the V-then-U convention used by
     # the plotting routines.
-    coords = names(coords)
-
```

Comment:
This was weird! I don't know why this was here. `names` effectively takes a list of coords and combines the names as a string, so
```names(['longitude', 'latitude'])```
would return the string `'longitude, latitude'` which would then mess up the processing below which expects coords to be a list.

Reply:
I did this, and I thought I'd undone it.  This was an attempt to fix errors when coords are passed in as integers.